src/main/java/nl/tudelft/contextproject/util/webinterface/ActionUtil.java

```diff
@@ -0,0 +1,115 @@
+package nl.tudelft.contextproject.util.webinterface;
+
+import nl.tudelft.contextproject.Main;
+import nl.tudelft.contextproject.model.entities.Bomb;
+import nl.tudelft.contextproject.model.entities.LandMine;
+import nl.tudelft.contextproject.model.entities.Pitfall;
+
+/**
+ * Utility to perform actions generated by the web client.
+ */
+public final class ActionUtil {
+
+	/**
+	 * Private constructor to avoid initialization.
+	 */
+	private ActionUtil() {}
+
+	/**
+	 * Perform an action.
+	 *
+	 * @param action
+	 * 		the action to perform
+	 * @param xCoord
+	 * 		the x coordinate to perform the action on
+	 * @param yCoord
+	 * 		the y coordinate to perform the action on
+	 */
+	public static void perform(String action, int xCoord, int yCoord) {
+		switch (action) {
+			case "placebomb":
+				placeBomb(xCoord, yCoord);
+				break;
+			case "placepitfall":
+				placePitfall(xCoord, yCoord);
+				break;
+			case "placemine":
+				placeMine(xCoord, yCoord);
+				break;
+			case "spawnenemy":
+				spawnEnemy(xCoord, yCoord);
+				break;
+			case "dropbait":
+				dropBait(xCoord, yCoord);
+				break;
+			default:

```

Comment:
This entire class isn't tested yet as most of the functionality isn't there yet. All tests will be written once all methods have been added.